src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -482,9 +498,9 @@ COMMENT ON COLUMN defendant.cas_id
 IS 'foreign key from court_case';
 
 CREATE TABLE device_register
-(der_id                      INTEGER                       NOT NULL
+(node_id                    INTEGER                       NOT NULL

```

Comment:
Are we missing a sequence for this table? It looks like @karen-hedges was expecting 
`CREATE SEQUENCE nod_seq START WITH 10000;`

https://github.com/hmcts/darts-api/pull/359/files#r1304035986